package.json

```diff
@@ -158,6 +158,12 @@
           "default": {},
           "description": "Specify the list of additional environment variables used for running tests."
         },
+        "mesonbuild.testJobs": {
+          "type": "integer",
+          "default": -1,
+          "minimum": -1,
+          "description": "Specify the maximum number of tests executed in parallel. -1 for number of CPUs, 0 for unlimited."
+        },
```

Comment:
Instead of magic integers can we use strings to indicate unlimited and ncpus?

Reply:
the "=0 means unlimited and =-1 means nproc" mimics semantics in ninja and most other tools with a `-j` option.
I don't have any strong feelings in either direction, but I don't think you can do "choice of two specific strings, or a number field" as a vscode setting?

Reply:
Your logic makes sense. Let's keep it.